lib/tasks/test_replication.rake

```diff
@@ -55,16 +58,27 @@ class EvmTestSetupReplication
 
   private
 
+  def fetch_branch
+    if system("git fetch --depth=1 https://github.com/ManageIQ/manageiq.git refs/heads/darga:#{TEST_BRANCH}")
+      return true
+    end
+    system("git fetch --depth=1 git@github.com:ManageIQ/manageiq.git refs/heads/darga:#{TEST_BRANCH}")
```

Comment:
Why twice?  I see one is http, and the other is git based, but http should be sufficient